src/sensors/lorreconstruction.rs

```diff
@@ -238,12 +238,11 @@ fn azimuthal_angle(hits: &[MCQT], xyzs: &SensorMap) -> Option<Vec<Angle>> {
         let &(x, y, _) = xyzs.get(&sensor_id)?;
         phis.push(y.atan2(x));
     }
+    // Maybe better to use the physical constants for pi/2 instead of turn(-0.25)
```

Comment:
I doubt it.

Also, no need to use `physical_constants` if all you want is variations on the theme of π, as these are all in the standard library: https://doc.rust-lang.org/std/f32/consts/index.html.

Reply:
Yeah, I found them without the `physical_constants` crate but I think it had a similar name. Anyway, I think I'll remove the commits that added `physical_constants` doesn't seem necessary for our application.